addon-test-support/setup-middleware-reporter.ts

```diff
@@ -42,16 +51,23 @@ export async function middlewareReporter(axeResults: AxeResults) {
       testName,
       urls: [],
       routes: [],
-      helpers: testMetaData.usedHelpers,
+      helpers: [],
       stack,
       violations: [],
     };
 
     currentViolationsMap = new Map();
+    currentUrls = new Set();
+    currentRouteNames = new Set();
+    currentTestHelpers = new Set();
   }
 
-  currentUrls.add(currentURL());
-  currentRouteNames.add(currentRouteName());
+  currentTestHelpers = new Set([
+    ...currentTestHelpers!.values(),
+    ...testMetaData.usedHelpers,
```

Comment:
If we stash off the `testMetaData` (into `currentTestResult`), we can avoid using `currentTestHelpers` at all here and instead update `pushTestResult` to grab the `usedHelpers` at the end

Reply:
Good call. Will change that.